ci/nits.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+#
+# Project nits enforced here
+#
+set -e
+
+cd "$(dirname "$0")/.."
+source ci/_
+
+# please don't print from --lib...
+declare prints=(
+  'print!'
+  'println!'
+  'eprint!'
+  'eprintln!'
+)
+
+if _ git grep "${prints[@]/#/-e }" src
```

Comment:
nit for `nits.sh`:
```bash
if _ git grep "${prints[@]/#/-e }" src; then
```

Reply:
Also how about checking `*/src` in addition to `src`

Reply:
most, I think.